server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/VacationResponseGetMethod.scala

```diff
@@ -70,13 +70,19 @@ class VacationResponseGetMethod @Inject() (serializer: Serializer,
         .flatMap(_ => asVacationResponseGetRequest(invocation.arguments))
         .fold(e => handleRequestValidationErrors(e, invocation.methodCallId),
           vacationResponseGetRequest =>
-            getVacationResponse(vacationResponseGetRequest, processingContext, mailboxSession)
-              .reduce(VacationResponseGetResult.empty, VacationResponseGetResult.merge)
-              .map(vacationResult => vacationResult.asResponse(vacationResponseGetRequest.accountId))
-              .map(vacationResponseGetResponse => Invocation(
-                methodName = methodName,
-                arguments = Arguments(serializer.serialize(vacationResponseGetResponse).as[JsObject]),
-                methodCallId = invocation.methodCallId))))
+            vacationResponseGetRequest.properties match {
+              case Some(properties) if !properties.asSetOfString.subsetOf(VacationResponse.allProperties) =>
+                SMono.just(Invocation.error(errorCode = ErrorCode.InvalidArguments,
+                  description = s"The following properties [${properties.asSetOfString.diff(VacationResponse.allProperties).mkString(", ")}] do not exist.",
```

Comment:
can we remove `{}`?

Reply:
No, as we don't use only the variable but methods from it, so the `${}` structure is necessary?